scripts/apps/archive/constants.ts

```diff
@@ -0,0 +1,24 @@
+
+export const PUBLISHED = 'published';
+export const SCHEDULED = 'scheduled';
+export const CORRECTED = 'corrected';
+
+export const KILLED = 'killed';
+export const RECALLED = 'recalled';
+export const UNPUBLISHED = 'unpublished';
+
+export const PUBLISH_STATES = [
```

Comment:
Why `PUBLISH_STATES` include `KILLED_STATES`? Wouldn't `ALL_STATES` be a better name in such case?

Reply:
there are other states before items is published, those publish states are all states when something was sent to subscribers

Reply:
Can you then add the remaining states or a comment with a link to code or docs with all possible states listed?